Packages/com.unity.inputsystem/InputSystem/InputSystemObject.cs

```diff
@@ -40,6 +42,50 @@ public void ReviveAfterDomainReload()
             SetUpRemoting();
         }
 
+        private SerializedObject GetPlayerSerializedPlayerSettings()
+        {
+            PlayerSettings[] array = Resources.FindObjectsOfTypeAll<PlayerSettings>();
```

Comment:
Use var. Could also use FirstOrDefault here. Okay to use Linq in editor code. With FirstOrDefault, this method becomes a single line of code.